Formula/docx2txt.rb

```diff
@@ -0,0 +1,19 @@
+class Docx2txt < Formula
+  desc "Converts Microsoft Office docx documents to equivalent text documents"
+  homepage "http://docx2txt.sourceforge.net/"
+  url "http://downloads.sourceforge.net/docx2txt/docx2txt-1.4.tgz"
+  sha256 "b297752910a404c1435e703d5aedb4571222bd759fa316c86ad8c8bbe58c6d1b"
+
+  depends_on "perl"
```

Comment:
Please don't hardcode our `perl`. If it needs a version of Perl a release of OS X doesn't ship use:

```ruby
depends_on :perl => "5.14" if MacOS.version <= :mavericks
```
For example, bending that style of syntax to your needs. If it's fine with OS X's shipped `perl` on OS X as far as you know, you can just remove the dependency completely.